src/spirv/spirv_module.h

```diff
@@ -3,9 +3,30 @@
 #include <unordered_set>
```

Comment:
```suggestion
#include <optional>
#include <unordered_set>
```

This is needed for it to compile on CI and my personal machine.